apps/frontend/src/pages/auth/libs/components/sign-in-form/sign-in-form.tsx

```diff
@@ -15,6 +16,7 @@ type Properties = {
 const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	const { control, errors, handleSubmit } = useAppForm<UserSignInRequestDto>({
 		defaultValues: DEFAULT_SIGN_IN_PAYLOAD,
+		mode: "onChange",
```

Comment:
Shouldn't be `onChange` mode

Reply:
But, in the requirements document we should make to real time validation